socialmedia.md

```diff
@@ -1,20 +1,118 @@
+* Facebook: __Add Here__
+* Twitter: __Add Here__
+* Instagram: __Add Here__
+* LinkedIn: __Add Here__
+* Snapchat: __Add Here__
+* Pinterest: __Add Here__
+* Youtube: __Add Here__
+
+### General Rules and Restrictions
+
+#### Posting on Social Media
+* Content will be posted to official school pages only.
+* There will be no content posted to a private page or closed group.
+* All posts are considered public, regardless of where they are posted, and should be treated as such.
+* The original post must be posted on a school page or account, but can then be shared by others from this place.
+* Any photographs or videos taken within school grounds, or during school time, should only be shared to a school account. 
+
+#### Photographs and Video
+* Media will be posted to school accounts only.
+* If the school keep a list of children who do not have parental consent to be online, this should be cross referenced before every post (and an audit trail of this kept for future reference).
+*	All children should be made aware a photo/video is being taken.
+*	We have our own “photo word” which must be said at all times before a photo is taken. This word is iconic to our school and is used to ensure all children know a photo is going to be taken. Our photo word is __“Swan”__.
+*	Children are always shown the photo after it is taken.
+*	There will be no media uploaded which contains one single child.
+*	There will be no media uploaded which contains identifiable features, such as name badges, name stickers on books, and so on.
+*	Employees will not post anything which may bring the school into disrepute or portray the school in a negative light.
+
+#### Engaging with Parents and Pupils
+*	Employees will not engage with parents or pupils via a closed or private account.
+*	All interactions must happen publicly and through school official channels only.
+*	Staff are encouraged to engage and reply where possible to increase engagement with others, except in the case where children are using a site which is not age-appropriate for them.
+
+#### Passwords
+*	Passwords to school social networking accounts should remain safe and secure at all times. For accountability, passwords are not designed to be shared between employees.
+*	Social Networks will be managed either through a single person or via a social media management solution in which all users will have their own log in and password. This is to ensure that full accountability exists.
+
+#### Permissions
+*	Unless stated otherwise, all media content obtained by employees remains the employees content. Employees may grant the school access to the content for distribution on social networks.
+*	All media uploaded must be owned by the school before it can be uploaded.
+
+#### Managing Social Networks
+*	We have assigned our social media specialist as __Mr Anonymous__.
+* We endeavour to ensure all content is moderated to ensure consistency and to ensure our posts paint the school in a positive light. 
```

Comment:
Staff should ensure that content is checked by another member of the team before posting, to ensure the content is abiding by the details in this policy.